src/syrupy/assertion.py

```diff
@@ -15,14 +14,12 @@ def __init__(
         self,
         *,
         update_snapshots: bool,
-        io_class: Type[SnapshotIO],
         serializer_class: Type[SnapshotSerializer],
         test_location: TestLocation,
         session,
     ):
         self._update_snapshots = update_snapshots
-        self._io_class = io_class
-        self._serializer_class = serializer_class
+        self._io_class = serializer_class
```

Comment:
And all the other renames
```suggestion
        self._serializer_class = serializer_class
```